src/mixins/date-mixin.ts

```diff
@@ -76,14 +76,17 @@ export default class DateMixin extends Vue {
     // safety check
     if (!isDate(date) || isNaN(date.getTime())) return null
 
-    const dateStr = date.toLocaleDateString('en-CA', {
+    // NB: some versions of Node have only en-US locale
+    // so use that and convert results accordingly
+    const dateStr = date.toLocaleDateString('en-US', {
```

Comment:
I needed to do this because I'm using Node 12 locally, and a bunch of unit tests that compared dates failed. This is the fix (and also line 157, below).

Ref: https://github.com/nodejs/node/issues/8500